docs/pages/deploy-a-cluster/aws-gslb-proxy-peering-ha-deployment.mdx

```diff
@@ -0,0 +1,252 @@
+
+### Teleport Proxy DNS records for GSLB:
+|Record name|Type|Routing Policy|Region|Value|
+|---|---|---|---|---|
+|```teleport.example.com```|CNAME|Latency|us-west-1| ```elb.us-west-1.amazonaws.com``` |
+|```*.teleport.example.com```|CNAME|Latency|us-west-1| ```elb.us-west-1.amazonaws.com``` |
+|```teleport.example.com```|CNAME|Latency|eu-central-1| ```elb.eu_central-1.amazonaws.com```|
+|```*.teleport.example.com```|CNAME|Latency|eu-central-1| ```elb.eu_central-1.amazonaws.com```|
+
+<Admonition title="Required permissions">
+
+If you are using Let's Encrypt to provide TLS credentials to your Teleport
+instances, the TLS credential system we mentioned earlier needs permissions to
+manage Route53 DNS records in order to satisfy Let's Encrypt's DNS-01 challenge. 
+
+</Admonition>
+
+### Teleport resource agent configuration for GSLB
+To facilitate latency routing, resource agents  must be configured to point ```proxy_server:``` to 
+the GSLB domain configured in Route53 _not the specific proxy NLB address_.
+
+For example:
+
+```
+version: v3
+teleport:
+    nodename: ssh-node
+    ...
+    proxy_server: teleport.example.com:443
+    ...
+    ssh_service:
+        enabled: yes
+    ...
+```
+Review the [configuration reference](https://goteleport.com/docs/reference/config/) page for 
+additional settings.       
+
+## Configure Proxy Peering
+
+In this deployment architecure, [Proxy Peering](https://goteleport.com/docs/architecture/proxy-peering/) is used to restrict the number of connections made from 
+resources to proxies in the Teleport Cluster.
+
+This guide covers the necessary Proxy Peering settings for deploying an HA Teleport Cluster routing resource
+traffic with GSLB. 
+
+### Auth Service Proxy Peering configuration 
+
+The Teleport Auth Service must be configured to use the proxy_peering tunnel strategy as shown in the example below:
+
+```
+auth_service:
+ ...
+ tunnel_strategy:
+  type: proxy_peering
+  agent_connection_count: 2
+```
+Reference the [Auth Server configuration](https://goteleport.com/docs/reference/config/#auth-service) reference page 
+for additional settings.
+
+### Proxy Service Proxy Peering configuration 
+
+Proxies must advertise a peer address which can be configured to use one of the two options listed below:
+
+**Option 1:** You can set peer_public_addr: to the specific name of that proxy. This is the recommended 
+method for lowest latency and most reliable connection.
+
+```
+version: v3
+teleport:
+...
+proxy_service:
+  ...
+  peer_public_addr: teleport.example.com:3021
+  ...
+```
+
+**Option 2:** Proxies can use peer_public_addr: to advertise the proxy NLB. When using this method 
+you could incur additional latency or user experience issues because peer proxies must continually dial through the NLB until 
+they establish connection to the correct peer target.
+
+
+```
+version: v3
+teleport:
+...
+proxy_service:
+  ...
+  peer_public_addr: teleport-example-nlb-us-east-1.amazonaws.com:3021
+  ...
+```
```

Comment:
IMO we shouldn't suggest that people do this for the reasons I listed above. It's easier to configure, but will result in weird behaviour on restarts or in the event of flappy connectivity and make troubleshooting harder than it needs to be (is the lack of connectivity due to a timing problem, a load balancer problem, a problem on the individual proxy host etc)

Proxy peering is not designed to constantly connect and disconnect from the LB until it gets what needs like the tunnel/agent mesh is - it's slower than that and will take a lot longer to settle into a happy state.